tests/e2e/policy_driven_vol_allocation.go

```diff
@@ -75,6 +81,22 @@ var _ = ginkgo.Describe("[vol-allocation] Policy driven volume space allocation
 		}
 		govmomiClient := newClient(ctx, &e2eVSphere)
 		pc = newPbmClient(ctx, govmomiClient)
+
+		vcAddress := e2eVSphere.Config.Global.VCenterHostname
+		vcAdminPwd := GetAndExpectStringEnvVar(vcUIPwd)
+		govcCmd := "export GOVC_INSECURE=1;"
+		govcCmd += fmt.Sprintf("export GOVC_URL='https://administrator@vsphere.local:%s@%s';",
```

Comment:
this may not work always since VC password can have some special characters which can mess up things here, try using `GOVC_USER` and `GOVC_PASSWORD` variables to be on safer side

Reply:
Have made code changes to use Env variable instead